src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.Dummy.cs

```diff
@@ -44,6 +44,9 @@ public override Encoding Encoding
                 get { return Encoding.UTF8; }
             }
 
+            public override SourceHashAlgorithm ChecksumAlgorithm
+                => SourceHashAlgorithm.Sha256;
```

Comment:
```suggestion
                => SourceHashAlgorithmUtils.DefaultContentAlgorithm;
```

Seems like this should flow with our default vs. being hard coded to 256